src/main/java/org/eclipse/uprotocol/cloudevent/factory/CloudEventFactory.java

```diff
@@ -60,10 +50,10 @@ public interface CloudEventFactory {
      */
     static CloudEvent request(String applicationUriForRPC,
                               String serviceMethodUri,
-                              Any protoPayload,
+                              Optional<Any> protoPayload,
```

Comment:
It is not common to pass Optional as an argument, https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments

Reply:
Thanks for the feedback @mishap4 , this also means that UTransport API needs to change to not pass optional.

Reply:
done